docs/source/schema-delegation.md

```diff
@@ -193,8 +193,105 @@ Also provides the `info.mergeInfo.delegateToSchema` function discussed above.
 
 [Transforms](./schema-transforms.html) to apply to the query and results. Should be the same transforms that were used to transform the schema, if any. After transformation, `transformedSchema.transforms` contains the transforms that were applied.
 
+<h3 id="createRequest">createRequest</h3>
+
+The `createRequest` is a utility function for creating queries with multiple, aliased, roots and possible argument name collisions. The function should be called with these parameters:
+
+```js
+createRequest(
+  targetSchema: GraphQLSchema,
+  targetOperation: 'query' | 'mutation' | 'subscription',
+  roots: Array<OperationRootDefinition>,
+  documentInfo: GraphQLResolveInfo,
+  transforms?: Array<Transform>,
+): Request
+```
+
+where `OperationRootDefinition` is the following:
+```js
+type OperationRootDefinition = {
+  fieldName: string,
+  // string to rename the root fieldName as
+  alias?: string,
+  // args passed to the root field
+  args?: { [key: string]: any },
+  // contains the `fieldNodes` that will act as the root field's selection set
+  info?: GraphQLResolveInfo
+};
+```
+
+#### Example
+```js
+User: {
+  bookings(parent, args, context, info) {
+    const { document, variables } = createRequest(
+      subschema,
+      'query',
+      [
+        { fieldName: 'node', alias: 'booking1', args: { id: 'b1' }, info },
+        { fieldName: 'node', alias: 'booking2', args: { id: 'b2' }, info },
```

Comment:
General question: why not just construct a query that uses [alias syntax](http://graphql.org/learn/queries/#aliases)?
```gql
query QueryName {
  booking1: node(id: "b1") {...}
  booking2: node(id: "b2") {...}
}
```

Reply:
I mean that is what this is doing, just programmatically right? This lets users define a batch operation (with many roots) and send that via a `fetch` etc. 

If you just form the query from a string, you are missing all the other operations that the transforms do.